plugins/spi/SPIOutputTest.cpp

```diff
@@ -235,7 +239,7 @@ void SPIOutputTest::testIndividualLPD8806Control() {
   buffer.SetFromString("255,128,0,10,20,30");
   output.WriteDMX(buffer);
   data = backend.GetData(0, &length);
-  const uint8_t EXPECTED1[] = { 0xc0, 0xff, 0x80, 0x8a, 0x85, 0x8f, 0 };
+  const uint8_t EXPECTED1[] = { 0xc0, 0xFF, 0x80, 0x8a, 0x85, 0x8f, 0 };
```

Comment:
Sorry to nit pick, if we're doing that, can we capitalise the other hex in the tests too please.